pkg/auth/ctmap_authenticator.go

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package auth
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/cilium/cilium/pkg/endpointmanager"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+	"github.com/cilium/cilium/pkg/maps/ctmap"
+	"github.com/cilium/cilium/pkg/monitor"
+	"github.com/cilium/cilium/pkg/u8proto"
+)
+
+type ctMapAuthenticator struct {
+	endpointManager endpointmanager.EndpointsLookup
+}
+
+func newCtMapAuthenticator(endpointManager endpointmanager.EndpointsLookup) datapathAuthenticator {
+	return &ctMapAuthenticator{
+		endpointManager: endpointManager,
+	}
+}
+
+func (r *ctMapAuthenticator) markAuthenticated(dn *monitor.DropNotify, ci *monitor.ConnectionInfo, _ *authResult) error {
+	ep := r.endpointManager.LookupCiliumID(dn.Source)
+	if ep == nil {
+		// Maybe endpoint was deleted?
+		log.WithField(logfields.EndpointID, dn.Source).Debug("auth: Cannot find Endpoint")
+		return nil
```

Comment:
i assume the original intention of @jrajahalme was that we don't want to flood the log with warnings in such likely (?) scenarios where a pod (and therefore EP) gets deleted in the meantime.

Reply:
that's how i interpreted it.

currently, all other cases throw an `error` which gets logged as warning (instead of debug) - and authentication fails.

anyway - it seems as this `ctmap_authenticator` needs to be replaced with an `authmap_authenticator` in the near future anyway ;)